### PR TITLE
update setup instructions

### PR DESCRIPTION
**Pull request by EvanBoyle**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To hack on Pulumi, you'll need to get a development environment set up. You'll w
 You can easily get all required dependencies with brew and npm
 
 ```bash
-brew install node pipenv python@3 typescript yarn go@1.12 golangci/tap/golangci-lint
+brew install node pipenv python@3 typescript yarn go@1.13 golangci/tap/golangci-lint
```
Comment (pgavlin) [inline CONTRIBUTING.md]: I think this can just be `go`

**Comment by EvanBoyle** [comment]
@pgavlin you don't think we should pin the go version? I experienced problems with a 1.13 install when I started. I imagine something similar could happen with `brew install go` starts resolving to 1.14.

**Comment by pgavlin** [comment]
Pinning it is fine as long as that command works. Does `brew` have a `go@1.13` alias?

**Comment by EvanBoyle** [comment]
It does indeed https://formulae.brew.sh/formula/go

**Pull request merged**